gulpfile.js

```diff
@@ -55,6 +56,11 @@ function style() {
         .pipe(bytediff.stop((data) => formatByteMessage('cssnano', data)))
         .pipe(sourcemaps.write('.'))
         .pipe(gulp.dest(paths.styles.dest))
+        .pipe(bytediff.start())
```

Comment:
It would be better to split this into a separate task. Something like:

```
function gzip(){
    return (
        gulp.src(paths.styles.dest)
            .pipe(bytediff.start())
            .pipe(gzip({ append: false }))
            .pipe(bitediff.stop((data) => formatByteMessage('gzip', data)))
    )
}
```

We don't have to pipe the gzip files to a destination, we're just using the console output of bytediff.

Reply:
@kylejrp makes sense! Should we just call this at the beginning of `style()` or somewhere else?

Reply:
I've resolved this in my latest commits to your branch - take a look over and confirm that you're happy with my changes!